packages/boilerplate-generator/boilerplate-generator.js

```diff
@@ -27,12 +25,10 @@ Boilerplate = function (arch, manifest, options) {
 Boilerplate.prototype.toHTML = function (extraData) {
   var self = this;
 
-  if (! self.baseData || ! self.func)
+  if (! self.baseData || ! self.template)
     throw new Error('Boilerplate did not instantiate correctly.');
 
-  return  "<!DOCTYPE html>\n" +
-    Blaze.toHTML(Blaze.With(_.extend(self.baseData, extraData),
-                            self.func));
+  return  "<!DOCTYPE html>\n" + self.template(_.extend(self.baseData, extraData));
```

Comment:
yup, i'll change it to something like `_.extend(_.extend({}, self.baseData), extraData)`, or if it's safe to use ES6, `Object.assign({}, self.baseData, extraData)`